test/api/buffer/receivers/delete/expected-state.json

```diff
@@ -1,55 +0,0 @@
-{
-  "branches": [
```

Comment:
Rovnaky pripad ako test `exports/delete`, aj tu `receivers/delete` to pada na kontrole `files`, kedze test, hned ako subory vytvorí, tak receivers zmaze a skonci (tj. zmeny sa nestihnut prejavit), ... a opat, tu nas stav projektu nejak extra nezaujima.